templates/statistics/statistics.html.twig

```diff
@@ -4,8 +4,43 @@
 
 {% block content %}
 
-    <h1 class="page-title">{% trans %}Statistics{% endtrans %}</h1>
-
     {% include '/includes/flash.html.twig' %}
 
+    <h1 class="page-title">{% trans %}Statistics{% endtrans %}</h1>
+    <table class="table"
+           data-toggle="table"
+           data-search="true"
+           data-page-size=5
+           data-pagination="true"
+           data-page-list="[5, 10, 25, 50]">
+        <thead class="thead-dark thead-primary">
+        <tr>
+            <td></td>
+            <th colspan="3">{% trans %}Individual{% endtrans %}</th>
+            <th colspan="3">{% trans %}Company{% endtrans %}</th>
+            <th colspan="3">{% trans %}Partner{% endtrans %}</th>
+            <th colspan="3">{% trans %}Cup{% endtrans %}</th>
+            <th colspan="3">{% trans %}Total{% endtrans %}</th>
+        </tr>
+        <tr>
+            <td></td>
+            {% for i in range(1,5) %}
+            <th>{% trans %}NL{% endtrans %}</th>
+            <th>{% trans %}FR{% endtrans %}</th>
+            <th>{% trans %}Total{% endtrans %}</th>
+            {% endfor %}
+        </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>{% trans %}Started{% endtrans %}</td>
+                {% include '/includes/statistics_row.html.twig' with {'counts': startedCounts} %}
```

Comment:
@stejes indien mogelijk de include verhuizen naar een sub folder van de statistics folder. De algemene includes folder is voor generische zaken, dit is specifiek voor statistieken.